PacklinkPro/Block/Adminhtml/Content/AutoTest.php

```diff
@@ -2,65 +2,16 @@
 /**
  * @package    Packlink_PacklinkPro
  * @author     Packlink Shipping S.L.
- * @copyright  2019 Packlink
+ * @copyright  2020 Packlink
  */
 
 namespace Packlink\PacklinkPro\Block\Adminhtml\Content;
 
-use Magento\Backend\Block\Template;
-use Packlink\PacklinkPro\Bootstrap;
-use Packlink\PacklinkPro\Helper\UrlHelper;
-
 /**
  * Class AutoTest.
  *
  * @package Packlink\PacklinkPro\Block\Adminhtml\Content
  */
-class AutoTest extends Template
+class AutoTest extends Content
```

Comment:
It is needed because it represents the page itself as requested by Magento API. Magento identifies backend pages by classes extended from `Content` class.

Reply:
ok